mease_lab_to_nwb/convert_syntalos/convert_syntalos.py

```diff
@@ -1,27 +1,52 @@
 """Authors: Alessio Buccino, Cody Baker, Szonja Weigl, and Ben Dichter."""
 from pathlib import Path
+from isodate import duration_isoformat
+from datetime import timedelta
 
-from .syntalosnwbconverter import SyntalosNWBConverter
+from mease_lab_to_nwb import SyntalosNWBConverter
```

Comment:
```suggestion
from . import SyntalosNWBConverter
```

Reply:
Also, I was able to get relative imports to work on my system, but only for downstream components where the parent package is known (i.e., all the other interfaces in this repo that use relative imports). But the main script doesn't associate with a parent package, so trying to use a relative import here returns an error.

Shall I assume it's something specific to my system, and change this back to a relative import here, or keep the version that works for me?

Reply:
I guess it's fine here, since this is being run a script.

Ideally there would be a command-line interface and this would be run as a package, and the relative import should work in that case